packages/react-liveness/package.json

```diff
@@ -55,6 +56,7 @@
     "@smithy/fetch-http-handler": "^2.1.3",
     "@smithy/protocol-http": "^3.0.3",
     "@smithy/signature-v4": "2.1.4",
+    "@smithy/types": "^3.3.0",
```

Comment:
Type packages should not be included as direct (or peer) dependencies

Reply:
Can we turn off this rule for that file then? this import is used outside of testing

Reply:
Discussed offline and added a types file for these imports with an eslint disable.